client/components/product-expiration/test/index.js

```diff
@@ -36,7 +36,11 @@ describe( 'ProductExpiration', () => {
 	it( 'should return the renewal date in when the date is in the future', () => {
 		const date = moment( new Date( 2100, 10, 10 ) );
 		const wrapper = shallow(
-			<ProductExpiration expiryDateMoment={ date } translate={ translate } />
+			<ProductExpiration
+				expiryDateMoment={ date }
+				renewDateMoment={ date }
```

Comment:
It could be useful to have this test a scenario where `expiryDateMoment` and `renewDateMoment` are different dates.

Separately, it might be nice to add one new test case to cover the new "Expires on..." scenario added in this PR.

Reply:
Thanks. Implemented.